docs/commands/lint.md

```diff
@@ -20,221 +16,375 @@ openapi lint [--generate-ignore-file] [--help]
 openapi lint --version
 ```
 
-## Options
+Each option is discussed in detail below, and examples are also provided.
 
-Option                   | Type      | Required    | Default     | Description
--------------------------|:---------:|:------------:|:-----------:|------------
-`entrypoints`            | `array`   | no           | -           | Array of API definition filenames that need to be linted. See [the Entrypoints section](#entrypoints) for more options
-`--config`               | `string`  | no           | -           | Specify path to the [configuration file](#custom-configuration-file)
-`--extends`              | `array`   | no           | -           | [Extend a specific configuration](#extend-configuration) (defaults or config file settings)
-`--format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`
-`--generate-ignore-file` | `boolean` | no           | -           | [Generate ignore file](#generate-ignore-file)
-`--help`                 | `boolean` | no           | -           | Show help
-`--max-problems`         | `number`  | no           | 100         | Truncate output to display the specified [maximum number of problems](#max-problems)
-`--skip-preprocessor`    | `array`   | no           | -           | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--skip-rule`            | `array`   | no           | -           | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--version`              | `boolean` | no           | -           | Show version number
+## How to run the `lint` command
+In your terminal, cd to the root of your project directory, then type the command along with any of the options described below.
```

Comment:
issue: This seems out of order, or just confusing - options are listed above this section? But they're described below...and they're not described all in one place? Also, this section doesn't deliver on its title. It doesn't show how to run the command...

Reply:
suggestion: I think we should reconsider this new page structure. I'm totally OK with more specific, actionable headings for usage examples. But I think we're making the page confusing if we remove the whole list/table of all supported options. It's important for users to be able to see all supported options in one place. The "old" table was doing a good job of briefly describing each option and then linking to a specific section on how to use the option - maybe we can build on that and make it better?

Reply:
I think the table is very important.

Same with having consistent page structures. (comment above about: Introduction - Usage - Options - Examples)

Reply:
I will reverse-engineer as discussed. Hopefully the second offering will be better.